lib/site_encrypt/phoenix.ex

```diff
@@ -54,6 +54,23 @@ defmodule SiteEncrypt.Phoenix do
     end
   end
 
+  @doc """
+  Merges paths to key and certificates to the `:https` configuration of the endpoint config for Bandit.
+
+  This is the same as `configure_https/2` but returns in the format that Bandit
+  (https://github.com/mtrudel/bandit/) expects.
+  """
+  defmacro configure_https_bandit(config, https_opts \\ []) do
```

Comment:
I think you could hide the decision as to whether to configure Cowboy or Bandit from the user by looking at the value of the `:adapter` key in the passed config?

Reply:
I agree. If this decision can be made automatically, then it should be done by the lib.

Reply:
Ah, good idea. Added in 958d46a314620c5760fbd76e64110354ab65e32a